### ISSUE TITLE
Shaman is Missing Attack Sprite

### ISSUE THREAD

**Issue by Snayff** [Snayff/nqp2#127]
## The Bug
The Shaman is missing the attack sprite.

## Screenshots
![image](https://user-images.githubusercontent.com/23128268/131252685-3d9ce303-13c3-4254-ab8d-faf55c4267aa.png)

**Comment by devmgardner** [comment]
A cursory glance through the scripts at
`nqp2/scripts/scenes/combat/elements/entity.py`,
`nqp2/scripts/scenes/gallery/ui.py`, and
`nqp2/scripts/scenes/unit_data/ui.py`
shows what I think is the issue. It appears the shaman has an "other" action assigned instead of an "attack" action. I also don't see where the "other" action animation would be called, so I'm not sure if those sprites would appear.

I will say I'm not the most familiar with the repo and am just trying to lend a hand!

**Comment by snayff** [comment]
@devmgardner thank you very much for taking a look! Absolutely appreciated. 

It turns out that I had moved both the `other` sprites and the `attack` sprites into the `/other` folder.  A quick move of the attack sprites back to `/attack` and we're golden. 
![image](https://user-images.githubusercontent.com/23128268/131253873-cb786a80-edbf-434b-a17e-9954b8731d5e.png)

P.S. I got the sprites in a pack and some of them have extra animations beyond the default ones (`move`, `idle` etc.) and it is those that I have moved to `other`. Just in case I need them later.
